config/config.go

```diff
@@ -0,0 +1,62 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+)
+
+type Config struct {
+	MySQL *MySQLConfig
+	JWT   *JWTConfig
+}
+
+type MySQLConfig struct {
+	User     string
+	Password string
+	Host     string
+	Port     int
+	DB       string
+}
+
+type JWTConfig struct {
+	JWTSecret string
+}
+
+func NewConfig() *Config {
+	return &Config{
+		MySQL: NewMySQLConfig(),
+		JWT:   NewJWTConfig(),
+	}
+}
+
+func NewMySQLConfig() *MySQLConfig {
+	u := os.Getenv("MYSQL_USER")
+	pw := os.Getenv("MYSQL_PASSWORD")
```

Comment:
Is there any other way to protect the database password during the ci/cd process?

Reply:
I 'm a bit weak in DevOps like processes but: 
i. I would not put this information on the repository.
ii. I would use different credentials for different stages in development.
iii. In my understanding even if the credentials where just in a .env file in a server somewhere there is still a chance to be "seen" through a tool like Sentry in the log stack trace. I believe there are solutions provided by third party companies to keep this kind of credentials safe.